.github/workflows/nodejs.yml

```diff
@@ -32,3 +32,28 @@ jobs:
         run: npm run lint
       - name: Test
         run: npm test
+
+  platform-compatibility-test:
+    name: Test platform compatibility
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
```

Comment:
I guess I'd lean more towards just adding these to the main build matrix.  Why not?

Reply:
[this is how you do it](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategymatrixexclude)

basically, add `exclude` prop to `matrix`, and specify the OS/node-version pairs we don't want to run.